cpp/benchmarks/io/parquet/parquet_writer_chunks.cpp

```diff
@@ -26,6 +26,21 @@
 
 #include <nvbench/nvbench.cuh>
 
+std::string format_throughput(double bytes_per_second, int precision = 2)
```

Comment:
This could be applied to several benchmarks which I can do in this PR if folks are open to it.

Reply:
Ah, I did not realize this just adds the human-readable throughput, does not change the current output. Then, I think it would be nice to add this to cuIO benchmarks, should not impact automation.
Feel free to move format_throughput to a common location, at least as far as I'm concerned :)